subsys/usb/class/mass_storage.c

```diff
@@ -422,10 +422,26 @@ static void memoryRead(void)
 	}
 }
 
+static bool check_cbw_data_length(void)
+{
+	if (!cbw.DataLength) {
+		LOG_WRN("Zero length in CBW");
+		csw.Status = CSW_FAILED;
```

Comment:
call fail() instead?
Probably could also be in lines: [451](https://github.com/zephyrproject-rtos/zephyr/pull/23307/commits/7fdcdd8d2eec497231d0ba0f35cd19d91f448f23#diff-7726e7e0e855220e1de8c888a9b06c4eR451), [484](https://github.com/zephyrproject-rtos/zephyr/pull/23307/commits/7fdcdd8d2eec497231d0ba0f35cd19d91f448f23#diff-7726e7e0e855220e1de8c888a9b06c4eR484)

Reply:
Well this is explicitly for the case `Zero length in CBW` and I wanted not to mix it with the `fail()`.